app/components/GuideDescriptionCard/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ * Guide Description Card
+ *
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Stars from 'components/Stars';
+
+import {} from 'react-intl';
```

Comment:
Take this out, I need to remove this. it's for internationalization, multi language support. It's slowing everything down